tests/adapters/switches/dell10g_test.py

```diff
@@ -225,13 +247,14 @@ def test_get_vlan_with_no_name(self):
         assert_that(vlan.name, equal_to(None))
         assert_that(len(vlan.ips), equal_to(0))
 
-    def test_get_vlan_standard_with_name(self):
+    def test_get_vlan_with_line_break(self):
         self.mocked_ssh_client.should_receive("do").with_args("show vlan id 1000").once().ordered().and_return([
 
             "VLAN   Name                             Ports          Type",
             "-----  ---------------                  -------------  --------------",
             "1000   MyVlan                           Po27,          Static",
             "                                        Te1/0/2-4,",
+            "                                                  ",
```

Comment:
There's no line break to test here, and ssh_client will not return empty lines, so any mocking of ssh_client should never test with empty lines